src/utils/macros.js

```diff
@@ -58,8 +58,13 @@ module.exports = class Macros {
     if (!this.debugHelpers.module) {
       if (this.localDebugBindings.length > 0) {
         let importPath = this.localDebugBindings[0].findParent(p => p.isImportDeclaration());
+        if (importPath === null) {
+          // import declaration in question seems to have already been removed
```

Comment:
I noticed this in the wild, my guess is two babel plugins are both mutating the graph. Resulting in this outcome...

Reply:
Might be related to https://github.com/babel/ember-cli-babel/pull/286#discussion_r295541590

Reply:
FWIW, those two babel plugins both operate on separate import paths.

Reply:
This may be tripping up: https://github.com/babel/babel/blob/11fa2461cec930168fd43cfeda733aab056d8318/packages/babel-core/src/config/config-descriptors.js#L240